twitter_python/part_1_streaming_tweets/tweepy_streamer.py

```diff
@@ -2,9 +2,8 @@
 from tweepy.streaming import StreamListener
 from tweepy import OAuthHandler
 from tweepy import Stream
- 
-import twitter_credentials
- 
+import json
+
```

Comment:
pple will figure out that they have to put their API keys  on the place they get an error